apps/design-tokens-ui/README.md

```diff
@@ -0,0 +1,9 @@
+# Design Tokens UI
+
+App for visualizing and editing design tokens
+
+## Development
+
+1. Install npm dependencies: `yarn install`/`npm install`
```

Comment:
please remove the `yarn` part as we use only `npm` due to the `package-lock.json` file